apps/src/applab/applab.js

```diff
@@ -62,6 +59,9 @@ import {
   actions as jsDebugger,
 } from '../lib/tools/jsdebugger/redux';
 import JavaScriptModeErrorHandler from '../JavaScriptModeErrorHandler';
+import connectToMakerBoard from '../lib/kits/maker/connectToMakerBoard';
+import * as makerCommands from '../lib/kits/maker/commands';
+import * as makerDropletConfig from '../lib/kits/maker/dropletConfig';
```

Comment:
I've been running into this a lot lately with the multiple `commands.js`, `api.js` and `dropletConfig.js` files myself, so I've gotten in the habit of checking the parent directory when navigating to a file; so maybe that's why I feel ready to jump in on this.  Maybe one of us should RFC this in a near-future PR, and explain editor shortcuts for finding things in such a world.